pagopa/io-app
# chore(Cross): [IOAPPX-414] Add `NativeStackNavigator` to the Design System section

> [!important]
> This is a reproposal of the following PR, merged and then reverted after concerns about Android builds:
> * https://github.com/pagopa/io-app/pull/5424

## Short description
This PR adds the new `NativeStackNavigator` to the Design System section. This change enables a true native modal behaviour on iOS (including _pull down to dismiss_), other than a native navigation on both platforms.

## List of changes proposed in this pull request
- Add `@react-navigation/native-stack` dependency
- Update `react-native-screens` dependency with the latest version

## How to test
Launch the app, go to the **Design System** section and check if everything is OK.
